src/ORM/Table.php

```diff
@@ -2304,4 +2305,26 @@ public function __debugInfo()
             'connectionName' => $conn ? $conn->configName() : null
         ];
     }
+
+    /**
+     * Validates that a belongsToMany or hasMany assoc has at least one entry.
+     *
+     * @param mixed $value
+     * @return boolean
+     */
+    public function validateAtLeast($value, $expectedCount = 0, $operator = '>')
```

Comment:
Ah ok
In that case the method name gets confusing, `validateCount` might then make more sense.